libmultilabel/nn/metrics.py

```diff
@@ -45,6 +45,7 @@ class NDCG(Metric):
     https://scikit-learn.org/stable/modules/generated/sklearn.metrics.ndcg_score.html
     Please find the formal definition here:
     https://nlp.stanford.edu/IR-book/html/htmledition/evaluation-of-ranked-retrieval-results-1.html
+    The target has to be a binary tensor.
```

Comment:
add

We do not use NDCG in ?? because of ??

Reply:
I'm not pretty sure what should be filled in the ??. I might just delete these problematic illustrations to not confuse anyone.